.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,16 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [main]
```

Comment:
We don't have a branch named `main`, but we likely want this job to run on all branches.
```suggestion
```